Results.html

```diff
@@ -21,14 +22,14 @@
     <div class = "nav">
         <ul>
             <li> <a href="index.html"> About </a> </li>
-            <li> <a href="Results.html" class="selected"> Results </a> </li>
```

Comment:
I guess I don't understand the class name "selected" here.  Does that really make sense?  Is it "selected"?  Or is it highlighted, darkened, etc.   What describes the behavior?